docs/errors/E220.md

```diff
@@ -0,0 +1,13 @@
+# E220: Semicolon after else may cause unexpected execution of its body.
+
+A semicolon next to the `else` keyword can cause its body to be executed even when previous `if` statement condition is true.
+
+    if (true) {
+        console.log("true");
+    } else; {
+        console.log("& false");
+    }
+    > true
+    > & false
```

Comment:
Don't include command output in the example like this. It'll be interpreted as JavaScript code.

If you want to include the command output, make it a separate quotation paragraph:

```suggestion

> ```
> true
> & false
> ```
```

or make it a comment:

```suggestion
    // true
    // & false
```